src/status_im2/contexts/communities/menus/request_to_join/view.cljs

```diff
@@ -1,6 +1,7 @@
 (ns status-im2.contexts.communities.menus.request-to-join.view
   (:require [quo.react-native :as rn]
```

Comment:
please replace quo here as well as you're making changes in this ns

Reply:
Done!